models/topics.model.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../db/connection.js');
+
+exports.returnTopics = () => {
+    return db.query('SELECT * FROM topics').then(({rows}) => {
+        return rows
+    })
+    .catch((err) => {
+        throw err
+    })
```

Comment:
Your model is still part of your Promise chain. Any errors will be caught when passed back to the controller. You also want to be consistent and have all your catches in once place 🙂